ObedZac/iREPORTER
# Update get all redflags end point

# 162219852 get all red flags end point

### *What does this PR do*?

This PR creates the end point for the API to getting all red flags
the test for getting all red flags should pass

### *Description of Task to be completed*?

The app should get all red flags

### *How should this be manually tested*?

create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest and the test for getting all red flags should pass
or run the app via python run.py / flask run and test on postman

### *What are the relevant pivotal tracker stories*?

162219852
